src/js/models/v2/awardsV2/CoreAwardAgency.js

```diff
@@ -6,6 +6,7 @@
 const CoreAwardAgency = {
     toptierName: '--',
     populateCore(data) {
+        this.id = data.id || '--';
```

Comment:
When the agency `id` is unavailable, the link becomes `/#/agency/--`. I think it would make more sense to keep make `id` empty string or `null` and go to the agency landing page instead of a broken link.